packages/core/ios/Fabric/LottieAnimationViewComponentView.mm

```diff
@@ -0,0 +1,125 @@
+#import "LottieAnimationViewComponentView.h"
+
+#import <react/renderer/components/lottiereactnative/ComponentDescriptors.h>
+#import <react/renderer/components/lottiereactnative/EventEmitters.h>
+#import <react/renderer/components/lottiereactnative/Props.h>
+#import <react/renderer/components/lottiereactnative/RCTComponentViewHelpers.h>
+
+#import "RCTFabricComponentsPlugins.h"
+#import <React/RCTView.h>
+#import "LottieContainerView.h"
+#import "RNLottieFabricConversions.h"
+
+using namespace facebook::react;
+
+@interface LottieAnimationViewComponentView () <RCTLottieAnimationViewViewProtocol>
+@end
+
+@implementation LottieAnimationViewComponentView {
+    LottieContainerView *_view;
+}
+
+- (instancetype) initWithFrame:(CGRect)frame
+{
+    if (self = [super initWithFrame:frame]) {
+        static const auto defaultProps = std::make_shared<const LottieAnimationViewProps>();
+        _props = defaultProps;
+        
+        _view = [LottieContainerView new];
+        self.contentView = _view;
+    }
+    
+    return self;
+}
+
+#pragma mark - RCTComponentViewProtocol
+
++ (ComponentDescriptorProvider)componentDescriptorProvider
+{
+    return concreteComponentDescriptorProvider<LottieAnimationViewComponentDescriptor>();
+}
+
+- (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &)oldProps {
+    const auto &oldLottieProps = *std::static_pointer_cast<const LottieAnimationViewProps>(_props);
+    const auto &newLottieProps = *std::static_pointer_cast<const LottieAnimationViewProps>(props);
+    
+    if(oldLottieProps.resizeMode != newLottieProps.resizeMode) {
+        [_view setResizeMode:RCTNSStringFromString(newLottieProps.resizeMode)];
+    }
+    
+    if(oldLottieProps.sourceJson != newLottieProps.sourceJson) {
+        [_view setSourceJson:RCTNSStringFromString(newLottieProps.sourceJson)];
+    }
+    
+    if(oldLottieProps.sourceName != newLottieProps.sourceName) {
+        [_view setSourceName:RCTNSStringFromString(newLottieProps.sourceName)];
+    }
+    
+    if(oldLottieProps.sourceURL != newLottieProps.sourceURL) {
+        [_view setSourceURL:RCTNSStringFromString(newLottieProps.sourceURL)];
+    }
+    
+    if(oldLottieProps.progress != newLottieProps.progress) {
+        [_view setProgress:newLottieProps.progress];
+    }
+    
+    if(oldLottieProps.loop != newLottieProps.loop) {
+        [_view setLoop:newLottieProps.loop];
+    }
+    
+    if(oldLottieProps.speed != newLottieProps.speed) {
+        [_view setSpeed:newLottieProps.speed];
+    }
+    
+    if(oldLottieProps.colorFilters != newLottieProps.colorFilters) {
+        [_view setColorFilters:convertColorFilters(newLottieProps.colorFilters)];
+    }
+    
+    if(oldLottieProps.textFiltersIOS != newLottieProps.textFiltersIOS) {
+        [_view setTextFiltersIOS:convertTextFilters(newLottieProps.textFiltersIOS)];
+    }
```

Comment:
These comparisons work because of the struct operators set [above](https://github.com/lottie-react-native/lottie-react-native/pull/955/files#diff-7e1b40b8d45473fba9a66e1693cee215752e9f2d8f845d6dfd13059021529deaR13-R20)